aspnetcore/fundamentals/startup.md

```diff
@@ -1,75 +1,106 @@
-
-## Services Available in Startup
-
-ASP.NET Core dependency injection provides services during an application's startup. You can request these services by including the appropriate interface as a parameter on your `Startup` class's constructor or its `Configure` method. The `ConfigureServices` method only takes an `IServiceCollection` parameter (but any registered service can be retrieved from this collection, so additional parameters are not necessary).
+## Services available in Startup
 
-Below are some of the services typically requested by `Startup` methods:
+ASP.NET Core [dependency injection](xref:fundamentals/dependency-injection) provides services during an app's startup. These services are requested by including the appropriate interface as a parameter on the `Startup` class's constructor or its `Configure` method. The `ConfigureServices` method only takes an `IServiceCollection` parameter. Any registered service can be retrieved from this collection, so additional parameters aren't necessary.
 
-* In the constructor:  `IHostingEnvironment`, `ILogger<Startup>`
-* In `ConfigureServices`:  `IServiceCollection`
-* In `Configure`:  `IApplicationBuilder`, `IHostingEnvironment`, `ILoggerFactory`
+Some of the services typically requested by `Startup` methods:
 
-Any services added by the ``WebHostBuilder`` ``ConfigureServices`` method may be requested by the ``Startup`` class constructor or its ``Configure`` method. Use `WebHostBuilder` to provide any services you need during `Startup` methods.
+* In the constructor: `IHostingEnvironment`, `ILogger<Startup>`
+* In `ConfigureServices`: `IServiceCollection`
+* In `Configure`: `IApplicationBuilder`, `IHostingEnvironment`, `ILoggerFactory`
 
 ## The Configure method
 
-The `Configure` method is used to specify how the ASP.NET application will respond to HTTP requests. The request pipeline is configured by adding [middleware](middleware.md) components to an `IApplicationBuilder` instance that is provided by dependency injection.
+The [Configure](/dotnet/api/microsoft.aspnetcore.hosting.startupbase.configure) method is used to specify how the app responds to HTTP requests. The request pipeline is configured by adding [middleware](xref:fundamentals/middleware) components to an [IApplicationBuilder](/dotnet/api/microsoft.aspnetcore.builder.iapplicationbuilder) instance that's provided by [dependency injection](xref:fundamentals/dependency-injection).
 
-In the following example from the default web site template, several extension methods are used to configure the pipeline with support for [BrowserLink](http://vswebessentials.com/features/browserlink), error pages, static files, ASP.NET MVC, and Identity.
+A typical web app configures the pipeline with support for [BrowserLink](http://vswebessentials.com/features/browserlink), error pages, static files, ASP.NET MVC, and Identity:
 
 [!code-csharp[Main](../common/samples/WebApplication1/Startup.cs?highlight=8,9,10,14,17,19,21&start=58&end=84)]
```

Comment:
Update to match the current template:
https://github.com/aspnet/templating/blob/aba10e768b91b73be7c7166c745410eb30eb64bf/src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/Startup.cs#L83-L112

Reply:
Yep ... did it on the last commit.